requirements-dev.txt

```diff
@@ -1,4 +1,5 @@
 mock==2.0.0
+mypy==0.790
```

Comment:
Why is this new dependency required?

Reply:
because there is no specification of which mypy version for type hint check.

should i remove it?

Reply:
I wouldn't add it if we don't use it currently.